projects/organization-management/src/app/pages/users/user-budget/user-budget.component.ts

```diff
@@ -0,0 +1,39 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+
+import { Price } from 'ish-core/models/price/price.model';
+import { UserBudgets } from 'ish-core/models/user-budgets/user-budgets.model';
+
+@Component({
+  selector: 'ish-user-budget',
+  templateUrl: './user-budget.component.html',
+  changeDetection: ChangeDetectionStrategy.Default,
+})
+export class UserBudgetComponent implements OnInit {
+  @Input() budget: UserBudgets;
+
+  usedBudget: Price;
+
+  ngOnInit(): void {
+    if (this.budget?.budget) {
+      this.usedBudget = {
+        value: this.budget.budget.value - this.budget.remainingBudget.value,
+        currency: this.budget.budget.currency,
+        type: 'Money',
+      };
+    }
+  }
+
+  usedBudgetPercentage() {
```

Comment:
we can eventually use just one method with an argument

Reply:
There is a percent pipe for this 😉 
https://angular.io/api/common/PercentPipe